.circleci/config.yml

```diff
@@ -142,6 +150,14 @@ workflows:
               only: /.*/
           requires:
           - cli
+      - web_examples:
+          context: "NuCypher Tests"
+          filters:
+            tags:
+              only: /.*/
+          requires:
```

Comment:
I also think this is far too early in the workflow.  Let's move this closer to the other demo & example tests.